paasta_tools/mesos_tools.py

```diff
@@ -111,12 +111,16 @@ def is_mesos_leader(hostname=MY_HOSTNAME):
     return get_mesos_leader() == hostname
 
 
-def get_current_tasks(job_id):
+def get_current_tasks(job_id, include_orphans=False):
     """ Returns a list of all the tasks with a given job id.
     :param job_id: the job id of the tasks.
     :return tasks: a list of mesos.cli.Task.
     """
-    return get_mesos_master().tasks(fltr=job_id, active_only=False)
+    mesos_master = get_mesos_master()
+    framework_tasks = mesos_master.tasks(fltr=job_id, active_only=False)
+    if include_orphans:
+        framework_tasks += mesos_master.orphan_tasks()
```

Comment:
yeah I guess the filtering is ignored for orphans... I could add it in theory

I'm tempted to say we should ship this anyway just because this is a real bug that could cause an outage. That said I still haven't fully fixed the build...